### PR TITLE
Unnecessary code cleanup

### PR DESCRIPTION
**Pull request by ckelleyRH**
More code cleanup. Removing the annotations presents no functional change so is safe. Removing the private methods and field should also be safe.

### CODE REVIEW

File: base/common/src/test/java/com/netscape/cms/servlet/test/ConfigurationTest.java [hunk 1]
```diff
@@ -539,66 +522,4 @@ private static void constructTKSData(String host, String port, String pin, Strin
         cert5.setToken(CryptoUtil.INTERNAL_TOKEN_FULL_NAME);
         systemCerts.add(cert5);
     }
-
-    private static String generateCRMFRequest(CryptoToken token, String keysize, String subjectdn, boolean dualkey)
```
Comment (edewata) [inline base/common/src/test/java/com/netscape/cms/servlet/test/ConfigurationTest.java:543]: Could you restore this method for now? I'd like to convert it into a CLI. You can change it to a `public` method to get rid of the warning.

**Review by edewata — CHANGES_REQUESTED** [review]
Just one request, but otherwise it looks good.

File: base/common/src/test/java/com/netscape/cms/servlet/test/ConfigurationTest.java (hunk 1 above)
Comment (ckelleyRH) [reply base/common/src/test/java/com/netscape/cms/servlet/test/ConfigurationTest.java:543]: Done

**Review by edewata — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**